pkgs/development/tools/misc/texinfo/common.nix

```diff
@@ -113,9 +114,19 @@ stdenv.mkDerivation {
   ];
 
   nativeCheckInputs = [ procps ] ++ optionals stdenv.buildPlatform.isFreeBSD [ freebsd.locale ];
+  checkInputs = optionals (lib.versionAtLeast version "7.2") [ glibcLocales ];
 
   doCheck = interactive && !stdenv.hostPlatform.isDarwin && !stdenv.hostPlatform.isSunOS; # flaky
 
+  # musl does not support locales.
+  preCheck =
+    if interactive && stdenv.hostPlatform.isMusl then
```

Comment:
Any (dis)advantage of using `lib.optionalString (interactive && stdenv.hostPlatform.isMusl)` here instead of `if … then … else null`, @alyssais?

Reply:
Great forward thinking, @alyssais, looking forward to the clean up on staging